src/index.js

```diff
@@ -129,15 +130,37 @@ const App = () => {
 
  useEffect(() => {
   initChecks(); // Routine Initial Checks (onstartup)
+  setredirectOption(localStorage.getItem("redirectOption"));
```

Comment:
move `setredirectOption()` hook inside `initChecks();` to prevent reading localStorage when not needed i.e in scenarios like user in not authorized to use this app (no Auth) or in cases like Update screen or Maintenace screens are shown

Reply:
Cool I've moved setredirectOption() inside initChecks() 👍